src/_premake_init.lua

```diff
@@ -430,6 +430,16 @@
 		kind = "string",
 	}
 
+	api.register {
+		name = "fastuptodate",
+		scope = "project",
+		kind = "string",
```

Comment:
This should use `boolean`, we already have a concept of `On` and `Off`, it would be confusing to have a random set of APIs use `Enable`/`Disable`.